src/map/clif.c

```diff
@@ -2229,13 +2229,30 @@ void clif_addcards2(unsigned short *cards, struct item* item) {
 		cards[3] = item->card[i];
 }
 
+void clif_add_random_options(unsigned char* buf, struct item* item)
```

Comment:
Since this is a new function, could you add a doxygen documentation comment for it?

Please reword as you see fit:
```
/**
 * Fills in (what are those random options exactly btw?) from the given item into a buffer.
 *
 * Dummy datais used since this feature isn't supported yet (ITEM_RDM_OPT).
 * A maximum of 5 random options can be supported.
 *
 * @param buf[in,out] The buffer to write to. The pointer must be valid and initialized.
 * @param item[in]    The source item.
 */
```

You can then remove the comment inside the function, since it's included in the above.